config/_default/menus/menus.en.toml

```
@@ -1,54 +1,105 @@
-[[docs]]
+[[lotus]]
   name = "Getting started"
   weight = 10
-  identifier = "getting-started"
-  url = "getting-started"
+  identifier = "lotus-getting-started"
+  url = "/lotus/getting-started"
 
-[[docs]]
-  name = "Node Set-up"
-  weight = 15
-  identifier = "node-set-up"
-  url = "node-set-up"
-
-[[docs]]
-  name = "Developers"
-  weight = 17
-  identifier = "developers"
-  url = "developers"
-
-[[docs]]
-  name = "Storage providers"
+[[lotus]]
+  name = "Installation"
   weight = 20
-  identifier = "storage-providers"
-  url = "/storage-providers"
+  identifier = "lotus-installation"
+  url = "/lotus/installation"
 
-[[docs]]
-  name = "Tutorials"
+[[lotus]]
+  name = "Lotus Configuration"
   weight = 30
-  identifier = "tutorials"
-  url = "/tutorials"
+  identifier = "lotus-config"
+  url = "/lotus/lotus-config"
 
-[[docs]]
-  name = "APIs"
+[[lotus]]
+  name = "Node Management"
   weight = 40
-  identifier = "apis"
-  url = "/apis"
+  identifier = "lotus-node-management"
+  url = "/lotus/node-management"
 
-[[docs]]
+[[lotus]]
   name = "Project"
   weight = 50
   identifier = "project"
-  url = "/project"
+  url = "/lotus/project"
+
+[[storage-providers]]
+  name = "Getting Started"
+  weight = 10
+  identifier = "miner-init"
+  url = "/storage-providers/getting-started"
+
+[[storage-providers]]
+  name = "Setup and Configuration"
+  weight = 20
+  identifier = "lotus-miner-config"
+  url = "/storage-providers/lotus-miner-config"
+
+[[storage-providers]]
+  name = "Daily Operations"
+  weight = 30
+  identifier = "lotus-miner-operation"
+  url = "/storage-providers/lotus-miner-operation"
+
+[[storage-providers]]
+  name = "Seal Workers"
+  weight = 40
+  identifier = "seal-workers"
+  url = "/storage-providers/seal-workers"
+
+[[developers]]
+  name = "Developers"
+  weight = 10
+  identifier = "developers"
+  url = "/developers"
+
+[[tutorials]]
+  name = "Lotus Tutorials"
+  weight = 10
+  identifier = "lotus-tutorials"
+  url = "/tutorials/lotus-tutorials"
+
+[[tutorials]]
+  name = "Lotus Miner Tutorials"
+  weight = 20
+  identifier = "lotus-miner-tutorials"
+  url = "/tutorials/lotus-miner-tutorials"
 
 [[main]]
-  name = "Docs"
-  url = "/docs/set-up/install/"
+  name = "Lotus"
+  url = "/lotus/getting-started/what-is-lotus"
   weight = 10
 
-#[[main]]
-#  name = "Blog"
-#  url = "/blog/"
-#  weight = 20
+[[main]]
+  name = "Developers"
+  url = "/developers/api-access"
+  weight = 20
+
+[[main]]
+  name = "Storage Providers"
+  url = "/storage-providers/getting-started/overview"
+  weight = 30
+
+[[main]]
+  name = "Tutorials"
+  url = "/tutorials/lotus-tutorials/store-and-retrieve"
+  weight = 40
+
+[[main]]
+  name = "Knowledge Base"
+  url = "/kb"
+  identifier = "kb"
```

```suggestion
  url = "/kb"
  identifier = "kb"
```

Let's change this to `knowledge-base` on the off-chance that someone mistakes it for kilobytes.

---

@johnnymatthews Can we keep it for now? I will open another issue and work on changing this. I would prefer to get it merged as it is blocker for lotus.

---

Ok, sure.